src/DataStructures/LeviCivitaIterator.cpp

```diff
@@ -0,0 +1,65 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include "DataStructures/LeviCivitaIterator.hpp"
+
+#include "Utilities/Gsl.hpp"
+
+namespace {
+template <size_t Dim>
+constexpr void initialize_indexes_and_signs(
+    gsl::not_null<std::array<Index<Dim>, factorial(Dim)>*> indexes,
```

Comment:
Are you using `indexes` to differentiate from `indices` in some way? Because both are used in this PR